modules/installation-guide/partials/proc_importing-external-or-additional-tls-certificates.adoc

```diff
@@ -0,0 +1,218 @@
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+* You are ready to create `CheCluster` custom resource.
+
+.Procedure
+
+. Save the certificates you need to import, to a local file system.
++
+[CAUTION]
+====
+* Certificate files are typically stored as Base64 ASCII files, such as `.pem`, `.crt`, `.ca-bundle`. But, they can also be binary-encoded, for example, as `.cer` files. All Secrets that hold certificate files should use the Base64 ASCII certificate rather than the binary-encoded certificate.
+* {prod-short} already uses some reserved file names to automatically inject certificates into the ConfigMap, so you should avoid using the following reserved file names to save your certificates:
+  ** `ca-bundle.crt`
+  ** `ca.crt`
+====
+
+. Create a new ConfigMap with the required TLS certificates:
++
+[subs="+attributes,+quotes"]
+----
+$ {orch-cli} create configmap ca-certs --from-file=__<certificate-file-path>__ -n=__<{project-context}-namespace-name>__
+----
++
+To apply more than one certificate, add another `--from-file=_<certificate-file-path>_` option to the above command.
+
+. During the installation process, when creating the `CheCluster` custom resource, take care of configuring the right name for the created ConfigMap.
++
+====
+For a {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator] deployment,
+ensure you add the `spec.server.ServerTrustStoreConfigMapName` field with the name of the ConfigMap, to the `CheCluster` Custom Resource you will create during the installation:
+
+[source,yaml,subs="+quotes",options="nowrap",role=white-space-pre]
+----
+spec:
+  server:
+    ...
+    spec.server.ServerTrustStoreConfigMapName: ca-certs
+----
+====
+ifeval::["{project-context}" == "che"]
++
+====
+For a {prod-short} link:https://helm.sh/[Helm Chart] deployment, ensure you override the `global.tls.serverTrustStoreConfigMapName` Helm Chart property with the name of the ConfigMap when installing the {prod-short} Helm Chart. For this you should add the following arguments to the Helm command line:
+[subs="+quotes",options="nowrap",role=white-space-pre]
+----
+--set global.tls.serverTrustStoreConfigMapName=ca-certs
+----
+====
+endif::[]
+
+== On already-running {prod-short} installations
+
+.Prerequisites
+
+* The `{orch-cli}` tool is available.
+
+* You should first define the name of the ConfigMap you will use to import certificates:
++
+====
+On instances of {prod-short} deployed with the {prod-short} link:https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html[Operator],
+retrieve the name of the ConfigMap by reading the `spec.server.ServerTrustStoreConfigMapName` `CheCluster` Custom Resource property:
+
+[subs="+attributes,+quotes",options="nowrap",role=white-space-pre]
+----
+$ get checluster {prod-checluster} -n __<{project-context}-namespace-name>__ -o json-path --jsonpath={.spec.server.serverTrustStoreConfigMapName}
```

Comment:
```suggestion
$ {orch-cli} get checluster {prod-checluster} -n __<{project-context}-namespace-name>__ -o jsonpath={.spec.server.serverTrustStoreConfigMapName}
```

Reply:
Fixed in commit https://github.com/eclipse/che-docs/pull/1598/commits/3a2f0520b241716f87b2764864c4cfa7b8ac6fa2

Reply:
Fixed in commit https://github.com/eclipse/che-docs/pull/1598/commits/89c462e804f3d5d1b9df27933ffac38dca7b8250